### PR TITLE
chore: multi collateral comments

### PR DESCRIPTION
**Pull request by bayological**
### Description

This PR aims to improve the clarity of the sortedOracles contract's comments. Specifically, it clarifies that the contract stores rates for all mento collateral assets, rather than just CELO.

### Other changes

- N/A

### Tested

- N/A

### Related issues

- Closes https://github.com/mento-protocol/mento-general/issues/174

### Backwards compatibility

- N/A

### Documentation

- N/A

### CODE REVIEW

File: contracts/SortedOracles.sol [hunk 1]
```diff
@@ -255,18 +278,18 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
   }
 
   /**
-   * @notice Returns the number of rates.
-   * @param token The address of the token for which the CELO exchange rate is being reported.
-   * @return The number of reported oracle rates for `token`.
+   * @notice Returns the number of rates that are currently stored for a specifed rateFeedId.
+   * @param token The rateFeedId for which to retrieve the number of rates.
+   * @return uint256 The number of reported oracle rates storedn for the given rateFeedId.
```
Comment (philbow61) [inline contracts/SortedOracles.sol:283]:
```suggestion
   * @return uint256 The number of reported oracle rates stored for the given rateFeedId.
```

File: contracts/SortedOracles.sol [hunk 2]
```diff
@@ -255,18 +278,18 @@ contract SortedOracles is ISortedOracles, ICeloVersionedContract, Ownable, Initi
...
    */
   function numRates(address token) public view returns (uint256) {
     return rates[token].getNumElements();
   }
 
   /**
-   * @notice Returns the median rate.
-   * @param token The address of the token for which the CELO exchange rate is being reported.
-   * @return The median exchange rate for `token`.
+   * @notice Returns the median of the curretly stored rates for a specified rateFeedId.
```
Comment (philbow61) [inline contracts/SortedOracles.sol:290]:
```suggestion
   * @notice Returns the median of the currently stored rates for a specified rateFeedId.
```

**Review by philbow61 — CHANGES_REQUESTED** [review]
LGTM 
- some typos 
- what about the linting / slither warning?

**Review by philbow61 — APPROVED** [review]

**Review by ninocomputer — APPROVED** [review]

**Comment by bayological** [comment]
> LGTM
> 
> * some typos
> * what about the linting / slither warning?

Unfortunately, I can't update the slither db :(

**Pull request merged**